wakeonlan/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .wakeonlan import BROADCAST_IP
+from .wakeonlan import DEFAULT_PORT
+from .wakeonlan import create_magic_packet
+from .wakeonlan import send_magic_packet
+from .wakeonlan import main
```

Comment:
Should we expose the main function? I’m leaning towards no. That would mean the `wakeonlan` script in `pyproject.toml` needs to be updated.

Reply:
I'm also leaning towards no. I kept it here just to make sure not to break compatibility. Can't imagine anyone is calling `main()` from their script though.

Haven't been in a python environment for a while, but I think the idiomatic thing to do would be:

- rename to `_main`
- remove the `_main` import from `__init__.py`
- call `_main` from `__main__.py`
- repoint the script property in `pyproject.toml` to `__main__` (I think...)

Thoughts on that?

Reply:
I had to drop Python 3.6 support in order to get pytest working in CI, so this is already going to be a semver major release.

It appears `__main__.py` is used by [`zipapp`](https://docs.python.org/3/library/zipapp.html) to create executable zips. It looks nice, but also redundant for packages that consist of a single file. If we just keep `wakeonlan.py` as-is, people can use that instead.

Let’s just remove the `main` export here. :)